### ISSUE TITLE
Sync failed, after auth success starts auth proccess again.

### ISSUE THREAD

**Issue by 4-FLOSS-Free-Libre-Open-Source-Software** [walterpg/google-drive-sync#22]
**Describe the bug**
---------------------------
KeePass Sync for Google Drive
---------------------------
Fehler beim Senden der Anforderung.
---------------------------
OK   
---------------------------
![image](https://user-images.githubusercontent.com/46166740/97567159-f47df500-19e6-11eb-9af4-39449e4d1499.png)

**To Reproduce**
1. Go to 'Extras'
2. Click on 'Sync with Drive'
3. See: 
![image](https://user-images.githubusercontent.com/46166740/97567735-19726800-19e7-11eb-8da2-015b1934a8bb.png)
4. Authorise the Plugin in GDrive
5. See: Google Sign-In Authorization Complete
Please return to KeePass.
6. See Error: 
![image](https://user-images.githubusercontent.com/46166740/97567159-f47df500-19e6-11eb-9af4-39449e4d1499.png)

**Expected behavior**
database synchron 

**Screenshots**
see above
![image](https://user-images.githubusercontent.com/46166740/97572461-8df9d680-19e8-11eb-8f76-e89eb3526681.png)

**Please complete the following information:**
 - OS: Windows 10, Build 20h2
 - Browser Chrome
 - Version 4.0.3b
Keepass v 2.46

**Additional context**
The Plugin is successfully authorized drive access.
error translates: Error sending the request.

**Comment by walterpg** [comment]
Thanks for this report.

"KeePass GoogleSyncPlugin" as shown in your last screenshot is the original plugin (note the link to the project on SF.net).  So you are apparently using the legacy plugin's app credentials, as shown below.  Does your Sync Authorization configuration look something like this?

![image](https://user-images.githubusercontent.com/15221668/97714989-d8825e00-1a8f-11eb-94b9-b35fa830ab16.png)

When I use that option, Sign On doesn't let me authorize at all, as shown in the [Authorization Issues](https://www.kpsync.org/usage/authorize#authorization-issues) topic.  I highly recommend disabling the legacy credentials for the best experience.  Have you tried any of the options mentioned in the link?

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
Thanks for reply. I tried multiple times since. But still failed to set up.
> "KeePass GoogleSyncPlugin" as shown in your last screenshot is the original plugin (note the link to the project on [SF.net](//SF.net)). So you are apparently using the legacy plugin's app credentials, as shown below. Does your Sync Authorization configuration look something like this?
I tried to migrate to this fork. Prior used old plugin yes. So I was thinking the 3rd party app showing up in googles account was this plugin by mistake. I have now removed the access from the old plugin completely from the account.

- No, I did not enable any legacy options. My Sync Authorization configuration doesn't look like that.  Legacy credentials are disabled.

I did this steps:

![image](https://user-images.githubusercontent.com/46166740/99842880-e435e180-2b70-11eb-9820-d2d05362900e.png)
![image](https://user-images.githubusercontent.com/46166740/99842896-ec8e1c80-2b70-11eb-9383-3780e585d565.png)
![image](https://user-images.githubusercontent.com/46166740/99842904-f152d080-2b70-11eb-8491-946eecc8369e.png)
![image](https://user-images.githubusercontent.com/46166740/99842907-f3b52a80-2b70-11eb-9319-bbf9ea07ea22.png)
in my google account under "Apps with access to my account" no google-drive-sync appears anymore :(
